docusaurus.config.js

```diff
@@ -13,62 +13,102 @@ const cidRedirects = JSON.parse(fs.readFileSync('cid-redirects.json').toString()
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'Sumo Logic Docs',
-  tagline: '',
-  url: process.env.HOSTNAME || "http://localhost:3000",
-  trailingSlash: true,
   baseUrl: process.env.BASE_URL || "/",
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
   favicon: 'https://www.sumologic.com/favicon.ico',
   organizationName: 'sumologic', // Usually your GitHub org/user name.
   projectName: 'sumologic-documentation', // Usually your repo name.
-  stylesheets: [
-    'https://fonts.googleapis.com/css?family=Material+Icons',
-  ],
```

Comment:
Will removing this break the fonts anywhere else?

Reply:
It's not removed! It's down below, alphabetized.

Reply:
oh I see -- thank you!

Reply:
Hi @irontitan76 ! I super appreciate your work re-ordering this doc, and totally understand, but I think we will keep the order as-is. Aside from this, were there any other additions/deletions in this file you wanted to make?

Reply:
No problem at all, but is there a reasoning behind the order or lack thereof?

I'm a big proponent of code organization and standardization—probably one of the biggest ones around. In my opinion and although a small change, this alphabetization leads to an improved developer experience and simpler perusal of the configuration to find the value to update or verify. The maintainers of Docusaurus obviously don't agree hence the lack of organization here. That said, I'm not a maintainer of this repository, so I'll relent. 

As far as the changes, I don't believe I made any explicit ones here.

Reply:
I completely agree with your reasoning. But for this specific doc, the ordering aligns with the out-of-the-box, industry standard `docusaurus.config.js` sort order. As I've been working and collaborating in Docusaurus for years (and currently the sole maintainer), I can recite it in my sleep ha. I will take some of your re-ordering suggestions though!

You can check out some examples here:
https://github.com/ipavlic/apex-fp/blob/master/website/docusaurus.config.js
https://github.com/cryptodevhub/site/blob/main/docusaurus.config.js
https://github.com/ariga/atlas/blob/master/doc/website/docusaurus.config.js
https://github.com/webdriverio/webdriverio/blob/main/website/docusaurus.config.js